Agents/FenlandTrajectoryAgent/README.md

```diff
@@ -0,0 +1,152 @@
+
+docker login ghcr.io -u <github_username>
+<github_personal_access_token>
+
+&nbsp;
+## 1.2 Spinning up the stack
+
+Navigate to `Deploy/stacks/dynamic/stack-manager` and run the following command there from a bash terminal. To [spin up the stack], both a `postgis_password` and `geoserver_password` file need to be created in the `stack-manager/inputs/secrets/` directory (see detailed guidance following the provided link). There are several [common stack scripts] provided to manage the stack:
+
+bash
+# Start the stack (please note that this might take some time) - the port is optional and defaults to 3838
+bash ./stack.sh start <STACK_NAME> <PORT>
+
+# Stop the stack
+bash ./stack.sh stop <STACK_NAME>
+
+# Remove stack services (incl. volumes)
+bash ./stack.sh remove <STACK_NAME> -v
+
+
+After spinning up the stack, the GUI endpoints to the running containers can be accessed via Browser (i.e. adminer, blazegraph, ontop, geoserver). The endpoints and required log-in settings can be found in the [spin up the stack] readme.
+
+&nbsp;
+## 1.3 Deploying the agent to the stack
+
+As mentioned earlier, this agent requires [JPS_BASE_LIB] and [Stack-Clients] to be wrapped by [py4jps]. Therefore, after installation of all required packages (incl. `py4jps >= 1.0.32`), the `StackClients` resource needs to be added to allow for access through `py4jps`. All required steps are detailed in the [py4jps] documentation. However, the commands provided below shall suffice to compile the latest `StackClients` resource locally and install it inside the Docker container using the provided [Dockerfile]. If you encounter any error for spinning up the stack, please manually execute commands mentioned in Stack Client Settings section in this page.
+
+Simply execute the following command in the same folder as this `README` to build the required [Stack-Clients] resource and spin up the production version of the agent (from a bash terminal). The stack `<STACK NAME>` is the name of an already running stack.
+
+
+# Buildings the agent Docker image and pushing it
+bash ./stack.sh build
+
+# Deploying the agent (using pulled image)
+bash ./stack.sh start <STACK_NAME>
+
+
+&nbsp;
+# 2. Using the Agent
+ 
+## Provided functionality
+
+The agent will automatically register a task upon startup to assimilate the data in our target GPS folder into the KG. This background task can be triggered by an HTTP request after the agent starts up, accessible at http://localhost:5000/. 
+
+<center>-------- Data Instantiation ------------</center>
+
+- Post request to instantiate all electricity consumption/meter data to KG
+> /fenlandtrajectoryagent/process_and_instantiate
+
+
```

Comment:
Instructions missing on how to view the trajectories on TWA-VF 
1) How to access the visualisation? 
2) Is it populated on the data.json? (if yes, data.json is missing too)
3) Index.html is missing too

Reply:
I am unsure if this agent is part of a bigger stack, if yes it would be good to indicate where it is.

Else, it would be good to have a folder `stack-manager` that contains all the required config in spinning up the stack-manager, similar to what we did in 
[Pirmasens](https://github.com/cambridge-cares/TheWorldAvatar/tree/main/Deploy/stacks/Pirmasens)
[UK Flood Routing](https://github.com/cambridge-cares/TheWorldAvatar/tree/main/web/kingslynn-flood-routing)